src/main/java/duke/Deadlines.java

```diff
@@ -0,0 +1,47 @@
+package duke;
```

Comment:
Would it be better to separate out your classes into more packages, e.g duke.tasks, duke.exception?